plugins/services/src/js/filters/TasksRegionFilter.js

```diff
@@ -0,0 +1,54 @@
+import DSLFilterTypes from "#SRC/js/constants/DSLFilterTypes";
+import DSLFilter from "#SRC/js/structs/DSLFilter";
+import TaskUtil from "../utils/TaskUtil";
+
+const LABEL = "region";
+
+/**
+ * This filter handles the `region:XXXX` for tasks
+ */
+class TasksRegionFilter extends DSLFilter {
+  constructor(regions) {
+    super();
+    this.regions = [];
+
+    if (Array.isArray(regions)) {
```

Comment:
Are we expecting always Array? because otherwise what happens if regions aren't Array but the method get called still, that will probably throw an error.